routers/repo/release.go

```diff
@@ -247,6 +247,7 @@ func EditRelease(ctx *context.Context) {
 	ctx.Data["title"] = rel.Title
 	ctx.Data["content"] = rel.Note
 	ctx.Data["prerelease"] = rel.IsPrerelease
+	ctx.Data["draft"] = rel.IsDraft
```

Comment:
I'd prefer keeping commits focused on what they address. There are also still ALL_UPPER_CASE constants in that file, but would rather not touch now. Can change the IsDraft though, being a new one.